app/control.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package app
+
+import "encoding/binary"
+
+const (
+	ResizeMessage byte = iota + 1
+	DelayMessage
+)
+
+type Control struct {
+	Type           byte
+	Offset         int
+	DelayMs        uint16
+	TerminalWidth  uint16
+	TerminalHeight uint16
+}
+
+func (c Control) GetBytes() []byte {
```

Comment:
```suggestion
func (c Control) MarshalBinary() ([]byte, error) {
```

Reply:
I knew you are going to say that. it will complicate the calls:
```
						n, _ := controlRecorderBuffered.Write(controlMsg.MarshalBinary())
```
so I kind fo changed the name only, are you fine with that?